.node-version

```diff
@@ -0,0 +1 @@
+v18.17.0
```

Comment:
My issue with these are that we set the version in many places. Which means soon we'll have many places that disagree. The patch version needs updated constantly, so it's laborious to maintain. What behavior are you hoping to get out of these for development? Can it be done with less places to update and ideally set to something more like ^18.